auth_session_timeout/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# (c) 2015 ACSONE SA/NV, Dhinesh D
+# (c) 2015 ACSONE SA/NV, Dhinesh D, Jesse Morgan
```

Comment:
Please do not append to others' copyrights. Also, you did not change this file at all, so your claim to copyright on any aspect of it would be invalid.

Regarding the copyright headers & what happened here - we were significantly more lax on what happened with these headers previously. If you look at the history of this file, you'll see that Dhinesh D's copyright is actually invalid here too & I would argue it is invalid for the entire module. That argument isn't something to make for history though, but it's most certainly something to enforce going forwards.

Typically if you're going to claim copyright on a file in the header, you need to have written the whole thing. Otherwise, Git line-by-line attribution is an implicit copyright on things you do actually change.